src/js/profile/mobile/widget/Calendar.js

```diff
@@ -219,7 +326,7 @@
 				var self = this;
 
 				events.on(element, "vclick", self, false);
-			};
+			}
```

Comment:
Add semicolon to end of the function.
Need to sync all functions.

Reply:
done.